charts/rancher-monitoring/103.0.0+up40.1.2/templates/rancher-monitoring/hardened.yaml

```
@@ -131,18 +131,17 @@ spec:
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: monitoring-coredns-network-policy
+  name: rancher-monitoring-default-allow-all
```

The policy still only applies to the coredns pods (as the podSelector is still present) so the name should reflect this, instead of suggesting that it is a default that applies to all pods in the namespace.
```suggestion
  name: rancher-monitoring-coredns-allow-all
```